### ISSUE TITLE
Create Job content type on Grad College website

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#2862]
# Motivation

The current website has a job listing for Grad Assistantships and we need to replicate that functionality.

Listing page: https://www.grad.uiowa.edu/graduate-assistant-job-postings (No jobs are currently available).

On the existing site, these are called GA positions. People can log in with their HawkID and create these, so this content type will need to have permissions that allow for that on the new site.

You can see what this process looks like at: https://www.grad.uiowa.edu/ga-position (HawkID login required)

# Proposed solution
- [ ] Construct a content type based on the existing GA position. I will double check whether all the current fields are still needed, so check with me before starting this.
- [ ] Set permissions so that someone with the viewer role can create this content type, but not publish it.

**Comment by pyrello** [comment]
Dropping this here as a potential alternative to use webform to collect the submission and convert it into a node: https://www.drupal.org/project/webform_content_creator

**Comment by mjoneill** [comment]
Adding a couple of links in case it's important to make open jobs findable to people who might be looking for them: 
* https://schema.org/JobPosting
* https://developers.google.com/search/docs/data-types/job-posting

**Comment by pyrello** [comment]
This is on hold temporarily, pending some upcoming meetings that will decide the fate of this feature for the Grad College.

**Comment by pyrello** [comment]
Grad College is going to use Handshake for jobs, so this is no longer necessary.
